src/core_plugins/timelion/server/handlers/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,23 @@
+const parseSheet = require('../lib/parse_sheet');
+
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
```

Comment:
we should add tests for the change in white-space behavior too (not just for the new comma-separated behavior).